mvnw

```diff
@@ -0,0 +1,310 @@
+#!/bin/sh
```

Comment:
This is generated by the Takari plugin when someone runs the following:
```
mvn -N io.takari:maven:wrapper
```

It's just boilerplate code that provides cross-OS Maven support.